torch/onnx/_internal/fx/registration.py

```diff
@@ -175,3 +175,53 @@ def is_registered_op(
     def _all_registered_ops(self) -> Set[str]:
         """Returns the set of all registered function names."""
         return set(self._registry)
+
+
+@dataclasses.dataclass
```

Comment:
It can be frozen

Reply:
Actually if I put on forzen, do I still need @property?

Reply:
I don't think you will need property if the purpose is protecting the fields from being changed

Reply:
Done with frozen dataclass